src/__test__/index.test.ts

```diff
@@ -311,4 +316,36 @@ describe('graphqlify', () => {
       }
     `)
   })
+
+  // it('render GraphQL alias', () => {
```

Comment:
Could you remove these comments? Because `__params` API will change soon because of the following arguments:

https://github.com/acro5piano/typed-graphqlify/issues/2